### PR TITLE
Minor updates and fixes

### PR DESCRIPTION
**Pull request by acholyn**
Small bib tweaks - stop looking for bib-mention class and make `__str__` return full_citation. To align better with [docs](https://github.com/UCL-ARC/museum_of_dreams/wiki/Using-the-Admin-Interface) I've changed the editor header/footer colour so avoid confusion

### CODE REVIEW

**Review by tcouch — COMMENTED** [review]
One weakness here is that it doesn't allow for two Bibliography items to have the same short citation (since you always choose the first). If specifying the id as a data attribute in the tag doesn't work, maybe you need to set a unique constraint on the BibliographyItem's short citation field?

**Comment by acholyn** [comment]
Have updated it so we just select it by the pk - the template for how it inserts into the editor is a lot more restrictive than they let on so they are now inserted as a link with the href being the pk but we're still unable to add any classes or data attributes.

File: mod_app/static/admin/js/mentionsPluginConfig.js [hunk 1]
```diff
@@ -3,7 +3,8 @@
   document.addEventListener("DOMContentLoaded", function () {
     const itemTemplate =
       '<li data-id="{id}" data-value="{id}">{full_citation}</li>';
-    const outputTemplate = "<strong>{short_citation}</strong>";
+
+    const outputTemplate = '<a href="{id}" >{short_citation}</a>';
```
[the remark this answers is not in the document]
Comment (tcouch) [reply mod_app/static/admin/js/mentionsPluginConfig.js:7]: Sorry, I was just curious why attributes just disappear from the template.

[the remark this answers is not in the document]
Comment (acholyn) [reply mod_app/static/admin/js/mentionsPluginConfig.js]: it was a good find! it's just what I needed; it slipped my radar

**Pull request merged**